tests/tensor_search/test_get_stats.py

```diff
@@ -163,9 +163,9 @@ def test_long_text_splitting_vectors_count(self):
                 "normalize_embeddings": True,
                 "model": "random/small",
                 "text_preprocessing": {
```

Comment:
ditto RE `text_preprocessing` and  `normalize_embeddings`

Reply:
Yes, it will be fixed in another PR (created index accept `snake_case`)